### DISCUSSION TITLE
[Question] how do I download the new challenge images

### DISCUSSION THREAD

**Discussion by ninjadoc107** [QIN2DIM/hcaptcha-challenger#182] [General]
let's suppose that a new captcha comes, and I want to make a model for it, how do I download the new challenge images

**Comment by QIN2DIM** [comment]
If you can't describe your question in one sentence with a good issue title, you should ask it in the discussion.

**Comment by QIN2DIM** [comment]
If you know how the Hcaptcha-Challenger works, you won't ask.

**Comment by itschasa** [comment]
> let's suppose that a new captcha comes, and I want to make a model for it, how do I download the new challenge images

hcaptcha challenger isn't for downloading images, it's for solving captchas. if you want images, make a scraper. or if you want the models, download them from the releases tab.
